controller/createVNubanController.js

```diff
@@ -0,0 +1,198 @@
+import fetch from "node-fetch";
+import pool from "../configs/connection.js"
+import appConfig from '../configs/appConfig.js';
+import { getChampionsWithoutVnuban } from "../configs/queryConstants.js";
+import axios from 'axios';
+
+async function sendSqlFailureEmail(err) {
+  const emailBody = {
+    toEmails: [
+      'collections@maxdrive.ai', 
+      'props-email-notification@maxdrive.ai',
+    ],
+    subject: 'vNuban Creation Failure',
+    htmlBody: `<p>Hello Team, <br/> Database error for vNuban creation.</p>
+    <p>${err}</p>
+    <p>Regards</p>`
+  }
+ 
+  let url1 = `${appConfig.NOTIFICATION_PUSH_URL}/v1/email/send`
+  axios.post(url1, emailBody, {headers :{
+    'Content-Type': 'application/json',
+    'Authorization': `Bearer ${appConfig.authToken}`
+  }
+})
+  .then(function (response) {
+    //console.log(response);
```

Comment:
use async await to remove this callbacks. also remove commented lines. use try catch